src/scaffolding/wizard/ScaffoldDebuggingStep.ts

```diff
@@ -30,32 +30,38 @@ export class ScaffoldDebuggingStep extends AzureWizardExecuteStep<ScaffoldingWiz
         switch (wizardContext.platform) {
             case 'Node.js':
                 scaffoldContext.platform = 'node';
-                const nodeOptions: NodeScaffoldingOptions = {
-                    package: wizardContext.artifact,
-                };
-                await dockerDebugScaffoldingProvider.initializeNodeForDebugging(scaffoldContext, nodeOptions);
+                await dockerDebugScaffoldingProvider.initializeNodeForDebugging(

```

Comment:
Slightly more interesting change (here and below). ESLint doesn't like doing declarations inside a `case` (without using `{ }` blocking), because the variable is in scope throughout the `switch`, but is not actually assigned if and until the `case` it is in hits.